tools/automator/README.md

```diff
@@ -0,0 +1,116 @@
+# Automator
+
+## Description
+
+`automator.sh` is a bash script used to automate cross-repository workloads. It clones an user-defined list of GitHub repositories, runs a series of
+user-defined commands in each of those repositories, then forks, commits, and pull requests any changes back to the upstream repository. It can be executed
+either as a standalone script or in Prow CI as a presubmit, postsubmit, or peridoic job.
+
+## Prerequisites
+
+1. [Bash 4.0+] interpreter.
+
+1. The following third-party tools must be available in the execution environment:
+   - [`curl`]
+   - [`getopt`]
+   - [`git`]
+   - [`jq`]
+   - [`pr-creator`]
+   - [`realpath`]
+
```

Comment:
I'm not sure if we want to document anything additional, but I also needed to install `gnu-getopt` on the Mac the get things to work when running locally. The normal `getopt` did not work.